news/2025/2025-08-25-capture-the-achievement-2-concludes.md

```diff
@@ -0,0 +1,215 @@
+---
+layout: post
+title: Capture The Achievement 2 Concludes
```

Comment:
it'd be nicer if more unique titles would fit for these types of posts, but if anything, contests mostly use "results"

```suggestion
title: Capture The Achievement 2 Results
```